tests/legacy-cli/e2e/tests/commands/serve/preflight-request.ts

```diff
@@ -0,0 +1,15 @@
+import fetch from 'node-fetch';
+import { ngServe } from '../../../utils/project';
+
+export default async function () {
+  const port = await ngServe();
+  const { size, status } = await fetch(`http://localhost:${port}/main.js`, { method: 'OPTIONS' });
+
+  if (size !== 0) {
+    throw new Error(`Ecpected "size" to be "0" but got "${size}".`);
```

Comment:
```suggestion
    throw new Error(`Expected "size" to be "0" but got "${size}".`);
```
Same for the status check below.